scripts/configure-hypervisor.sh

```diff
@@ -373,6 +373,10 @@ generate_qemu_options() {
 		qemu_options+=(size:--disable-cloop)
 		qemu_options+=(size:--disable-dmg)
 		qemu_options+=(size:--disable-parallels)
+	fi
+
+
+	if [[ "${qemu_version_major}" -ge 5  && "${qemu_version_minor}" -ge 1 ) ]]; then
```

Comment:
no - this condition will be false for QEMU 6.0

Reply:
Eeek! good catch @devimc!!